src/services/salesforce/connection.ts

```diff
@@ -15,7 +24,31 @@ import {
 /**
  * Handles connections and requests to Salesforce org
  */
-export class SalesforceConnection extends Connection {}
+export class SalesforceConnection extends Connection {
+  public constructor(options: Connection.Options<Schema>) {
+    super(options);
+  }
+
+  async replaceClasses(sources: Map<string, string>) {
```

Comment:
There's still adding the retry calls or doing something to make this more likely to be stable, or we can leave until there is a problem.

Reply:
https://github.com/apex-dev-tools/benchmarker/issues/12